Numbers/unit_converter.py

```diff
@@ -2,27 +2,54 @@
 # 21EUR, 22UAH, 23USD ----> RUB
 # 31kg -> pound, 32pound -> kg
 
-def unit_converter(convert: int, value: int) -> str:
-    if convert == 11:
-        return f"{value} °C is equal to {(value*1.8)+32} °F"
-    elif convert == 12:
-        return f"{value} °F is equal to {(value-32)*0.5556} °C"
-    elif convert == 21:
-        return f"{value} EUR is equal to {value*86.93} RUB"
-    elif convert == 22:
-        return f"{value} UAH is equal to {value*2.72} RUB"
-    elif convert == 23:
-        return f"{value} USD is equal to {value*77.37} RUB"
-    elif convert == 31:
-        return f"{value} Pound is equal to {value*0.453592} kg"
-    elif convert == 32:
-        return f"{value} kg is equal to {value*2.2046} Pound"
-    else:
-        return f"I don't understand"
-
-
-print(unit_converter(11,12))
+def convert_unit(frm: str, to: str, value: int) -> str:
+    frm, to = frm.lower(), to.lower()
+    proper_combinations = [
+        ['celsius', 'fahrenheit'],
+        ['kg', 'pound']
+    ]
 
+    is_valid = False
+    for proper_comb in proper_combinations:
+        if frm and to in proper_comb and frm != to:
+            is_valid = True
+            break
 
+    if not is_valid:
+        return "invalid combination"
+
+    if frm == proper_combinations[0][0] and to == proper_combinations[0][1]:
+        return f"{value} °C is equal to {celsius_to_fahrenheit(value)} °F"
+
+    if frm == proper_combinations[0][1] and to == proper_combinations[0][0]:
+        return f"{value} °F is equal to {fahrenheit_to_celsius(value)} °C"
+
+    if frm == proper_combinations[1][0] and to == proper_combinations[1][1]:
+        return f"{value} Kg is equal to {kg_to_pound(value)} Pound"
+
+    if frm == proper_combinations[1][1] and to == proper_combinations[1][0]:
+        return f"{value} Pound is equal to {pound_to_kg(value)} Kg"
+
+
+def celsius_to_fahrenheit(val):
```

Comment:
Inline these functions into the code. As they are all one-liners and used only once, there's no point in making them separate functions.